extensions/src/poseHand/index.ts

```diff
@@ -0,0 +1,324 @@
+import { ArgumentType, BlockType, Extension, Block, DefineBlock, Environment, ExtensionMenuDisplayDetails, RuntimeEvent } from "$common";
+
+import Video from '../../../packages/scratch-vm/src/io/video';
+import * as handpose from '../../../packages/scratch-vm/node_modules/@tensorflow-models/handpose/dist';
+
+/**
+ * States the video sensing activity can be set to.
+ * @readonly
+ * @enum {string}
+ */
+const VideoState = {
```

Comment:
This is another thing we can consider stick into a file in `$common` or on the `Environment` object. like: https://github.com/mitmedialab/prg-extension-boilerplate/pull/223/files#r1085644932

Reply:
This is actually something I did for my own convenience so I wouldn't have to remember which numbers I assigned to correspond with turning the video on, off, or flipped. It's directly correlated with the `videoToggle` function/block and shouldn't be placed in a separate file unless the `videoToggle` function is also placed in that separate file

Reply:
We could also remove VideoState and replace everywhere where it's used with the respective numbers, but I think it's more clear what's going on if we keep it in

Reply:
Ah gotcha! Remind me to show you a cool typescript thing we can try tomorrow that I think could simplify this a little bit and give us more safety too!